doc/index.md

```diff
@@ -0,0 +1,31 @@
+(Tri-)Finger Robot Driver and Tools  {#mainpage}
+================================================
+
+
+This is the documentation of the `robot_fingers` package.
+
+The source code is hosted on
+[GitHub](https://github.com/open-dynamic-robot-initiative/robot_fingers).
+Please also use the issue system there if you have a question or want to report
+a bug.
+
+For more information, on the TriFinger robot and the general architecture of the
+software, see also our [paper](https://arxiv.org/abs/2008.03596) on the
+open-source version of the TriFinger robot.
+
+
+Content
+-------
+
+- @ref md_doc_installation
+- @ref md_doc_singularity
+- @ref md_doc_getting_started
+- @ref md_doc_hardware_testing
+
```

Comment:
can we point somewhere to the instructions how they have to set up their system (ludos people must have that somewhere). and also how to activate CAN etc.

Reply:
There doesn't seem to be proper documentation for this yet.  I suggest we add this later, so we can get the rest merged now.